src/packages/@ncigdc/containers/explore/ClinicalAggregations.tsx

```diff
@@ -526,7 +526,7 @@ const enhance = compose(
                                       ),
                                     )}
                                     style={{
-                                      order: customSorting[componentFacet.field] || 0,
+                                      order: customSorting(componentFacet.field) || 0,
```

Comment:
This calls the sorting order calculator, rather than just asking a dictionary.

Reply:
The order of all facets here is different from the order in line 461. Should change the asc in line 464.

Reply:
```suggestion
                                      order: customSorting(componentFacet.field),
```